timemachine/fe/atom_mapping.py

```diff
@@ -46,15 +46,19 @@ def mcs(
     # atoms
     params.AtomCompareParameters.CompleteRingsOnly = 1
     params.AtomCompareParameters.RingMatchesRingOnly = 1
-    params.AtomCompareParameters.matchValences = 0
+    params.AtomCompareParameters.MatchValences = 0
```

Comment:
good catch! fortunately it looks like we got lucky since the default `matchValences=False`